profile/README.md

```diff
@@ -0,0 +1,53 @@
+
+- [GCBM.Chile.Data_Processing](https://github.com/moja-global/GCBM.Chile.Data_Preprocessing): GCBM.Chile.Data_Processing is a repository of pre-processing scripts for the GCBM implementation in Chile.
+
+- [FLINT.Reporting](https://github.com/moja-global/FLINT.Reporting): FLINT.Reporting is a reporting tool for FLINT to classify FLINT output results into standard reporting formats.
+
+- [FLINT.JSON_Interface](https://github.com/moja-global/FLINT.JSON_Interface): FLINT.JSON_Interface is a JSON-based interface for FLINT to create and edit JSON files in an easy way.
+
+- [FLINT.Example](https://github.com/moja-global/FLINT.Example): FLINT.Example is a repository of example FLINT projects to help start building a project based on the FLINT framework.
+
+## How to involve?
+
+We accept contributions on all spheres of open source development life cycle. Here are quick links to get you started:
+
+- [Community Contributions](https://github.com/moja-global/About_moja_global/tree/master/Contributing#community-contributions)
+- [Code Contributions](https://github.com/moja-global/About_moja_global/tree/master/Contributing#code-contributions)
+- [Documentation Contributions](https://github.com/moja-global/About_moja_global/tree/master/Contributing#documentation-contributions)
+- [Management Contributions](https://github.com/moja-global/About_moja_global/tree/master/Contributing#management-contributions)
+
+## Contact
+
+As a developer you can use issues and discussions on the respective projects to report bugs, request features, and discuss changes. We recommend using the same to connect with maintainers, coaches and ambassadors. As a user, you can use the [moja.global](https://moja.global) website to connect with us.
+
+You can reach out to us through the following channels as well:
+
+- [info@moja.global](mailto:info@moja.global) : Ask questions about moja global relevant to the members and management with a point of focus for new user countries and policy makers.
+- [tsc@moja.global](mailto:tsc@moja.global) : Ask questions about contributing code, helping out with development, resolving issues and building new features with moja global teams.
+- [mentorship@moja.global](mailto:mentorship@moja.global) : Ask questions about participating in the community, sponsoring programs, funding opportunity and community development.
+- [Slack community](https://join.slack.com/t/mojaglobal/shared_invite/zt-o6ta1ug0-rVLjAo460~d7JbZ~HpFFtw ): Connect with other developers, scientists, policy makers and other interested people over our Slack community.
+
+We organize monthly Technical Steering Committee (TSC) meetings to discuss the future of moja global and its ecosystem. TSC meetings are open to all members of the community. We also conduct working group meetups every two weeks to discuss a specific area of interest.
```

Comment:
For the group meetings. Maybe the upcoming meeting's date, time, and link can be provided? So that it's open to all? Maybe?

Reply:
Hi @chicken-biryani 

Right now we don't have a public calendar. I will add this, once we have one. Thanks for the suggestion. I will make an issue for the same :+1: